src/epoch.rs

```diff
@@ -675,6 +704,33 @@ impl Epoch {
         self.as_gpst_duration().in_unit(Unit::Day)
     }
 
+    ///Returns the Duration since the UNIX epoch UTC midnight 01 Jan 1970.
+    pub fn as_unix_duration(self) -> Duration {
+        let cnt = self.get_num_leap_seconds();
+        // TAI = UNIX + leap_seconds + UNIX_OFFSET_UTC_SECONDS <=> UNIX = TAI - leap_seconds - UNIX_OFFSET_UTC_SECONDS
+        self.0 + i64::from(-cnt) * Unit::Second - UNIX_OFFSET_UTC_SECONDS * Unit::Second
```

Comment:
Would this be easier if it were defined as an offset from UTC. From what I saw, the UNIX epoch is `January 1st, 1970 at 00:00:00 UTC`. So I think something like the following would work (this isn't tested code so it's maybe wrong):

```suggestion
        self.as_utc_duration() - Self::from_gregorian_utc_at_midnight(1970, 1, 1)
```

That way the as_utc_duration function does all of the leap second calculations. (It also may be worth making that as_utc_duration function `pub`).

Reply:
Interesting idea. I'll test this approach.